drivers/mtd/mtd.c

```diff
@@ -138,23 +138,22 @@ int mtd_write(mtd_dev_t *mtd, const void *src, uint32_t addr, uint32_t count)
 }
 
 #ifdef MODULE_MTD_WRITE_PAGE
-int mtd_write_page(mtd_dev_t *mtd, const void *data, uint32_t page,
-                   uint32_t offset, uint32_t len)
+static size_t _write_sector(mtd_dev_t *mtd, const void *data, uint32_t sector,
+                            uint32_t offset, uint32_t len)
```

Comment:
Can you add some comments on what this function is supposed to do?

Reply:
I added a comment